### PR TITLE
Aswathy/feat: deriv prime page

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/deriv-prime/index.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { WithIntl } from 'components/localization/withIntl'
+import Layout from 'components/layout/layout'
+
+const DerivPrime = () => {
+    return (
+        <Layout type="partners" margin_top={10}>
```
Comment (hasan-deriv) [inline src/pages/partners/deriv-prime/index.tsx:7]: Why margin top 10?

File: src/pages/partners/deriv-prime/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { WithIntl } from 'components/localization/withIntl'
+import Layout from 'components/layout/layout'
```
Comment (mohsen-deriv) [inline src/pages/partners/deriv-prime/index.tsx:3]: please don't use this layout component, use the one we have in features folder, thanks :pray:

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: src/pages/partners/deriv-prime/index.tsx (hunk 2 above)
Comment (hasan-deriv) [inline src/pages/partners/deriv-prime/index.tsx:3]: We should go with atomic approach. This layout component is in old approach.

File: src/pages/partners/deriv-prime/index.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/pages/partners/deriv-prime/index.tsx:7]: I will remove it

File: src/pages/partners/deriv-prime/index.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/pages/partners/deriv-prime/index.tsx:3]: okay got it

File: src/pages/partners/deriv-prime/index.tsx [hunk 3]
```diff
@@ -1,13 +1,21 @@
 import React from 'react'
-import { WithIntl } from 'components/localization/withIntl'
 import Layout from 'components/layout/layout'
```
Comment (hasan-deriv) [inline src/pages/partners/deriv-prime/index.tsx:2]: `import Layout from 'features/components/templates/layout'` use this component..

File: src/pages/partners/deriv-prime/index.tsx [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
-        <Layout type="partners" margin_top={10}>
-            <></>
+        <Layout type="noNav">
+            <SEO
+                description="_t_With Deriv Prime's liquidity solutions for online trading, you get unbeatable spreads on the commodities, cryptocurrencies, and forex markets._t_"
+                title="_t_Liquidity solutions provider | Deriv Prime_t_"
+            />
+            <PaymentAgentAffiliateNav />
+            <DerivPrime />
         </Layout>
```
Comment (hasan-deriv) [inline src/pages/partners/deriv-prime/index.tsx]: Implement this inside **src/features/pages/partners/deriv-prime/index.tsx** here. And Call that component here. For, reference you can check the pages we already did by new approach.

**Review by hasan-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Comment by aswathy-deriv** [comment]
Please refer this PR -https://github.com/binary-com/deriv-com/pull/5418

**Pull request closed without merging**